multipath/multipath.conf

```diff
@@ -1,3 +1,11 @@
+# --- WARNING: DO NOT EDIT THIS FILE ---
+# The contents of this file may be overwritten at any future time through a
+# system update, causing any custom configuration to be lost.
+#
+# For custom multipath configuration, create a separate .conf file in the
+# /etc/multipath/conf.d/ directory.
+# --- END OF WARNING ---
```

Comment:
Maybe include the `Refer to "man multipath.conf"` bit here as well?

Reply:
Wouldn't this be a repetition of what line 11 already says?

If we replace the message with something telling users to modify custom.conf, then this will not be needed anymore. 

Else, maybe we can move line 11 up.